### PR TITLE
Random edit that needs approval

### PR DESCRIPTION
**Pull request by pkaminski**
This change is [Reviewable](https://reviewable-dev-piotr.firebaseapp.com/reviews/Reviewable/testing-public/9)

### CODE REVIEW

**Review by pkaminski — COMMENTED** [review]
> Reviewed 1 of 1 files at r1, all commit messages.
> *[Reviewable](http://localhost:8080/reviews/Reviewable/testing-public/9)* status: all files reviewed, all discussions resolved (waiting on @pkaminski)

File: README.md [hunk 1]
```diff
@@ -15,3 +15,5 @@ Fifth edit.
 Sixth edit.
 
 Seventh edit.
+
+Random edit.
```
Comment (pkaminski-test) [inline README.md:19]: I don't know about this.

**Review by pkaminski-test — CHANGES_REQUESTED** [review]

**Review by pkaminski — COMMENTED** [review]
> *[Reviewable](http://localhost:8080/reviews/Reviewable/testing-public/9)* status: all files reviewed, 1 unresolved discussion (waiting on @pkaminski-test)

___
*[`README.md` line 19 at r1](http://localhost:8080/reviews/Reviewable/testing-public/9#gh-1297759214:-Nc4LmGBCPUAP-MEF00I:b-896fix) ([raw file](https://github.com/Reviewable/testing-public/blob/b8a7352895ceb5942a268a6ef8d5eaad7bcb256e/README.md#L19)):*
<details><summary><i>Previously, pkaminski-test wrote…</i></summary><blockquote>

I don't know about this.
</blockquote></details>

Done.

File: README.md (hunk 1 above)
Comment (pkaminski-test) [reply README.md:19]: OK then.

**Review by pkaminski-test — APPROVED** [review]

**Pull request merged**